insights/parsers/systemctl_status_all.py

```diff
@@ -28,18 +32,52 @@ class SystemctlStatusAll(LogFileOutput):
                    | `-user-1000.slice
                    |   |-user@1000.service
 
-        * proc-sys-fs-binfmt_misc.automount - Arbitrary Executable File Formats File System Automount Point
-           Loaded: loaded (/usr/lib/systemd/system/proc-sys-fs-binfmt_misc.automount; static; vendor preset: disabled)
-           Active: active (running) since Thu 2021-09-23 12:03:43 UTC; 3h 7min ago
-            Where: /proc/sys/fs/binfmt_misc
-             Docs: https://www.kernel.org/doc/html/latest/admin-guide/binfmt-misc.html
-                   https://www.freedesktop.org/wiki/Software/systemd/APIFileSystems
-
         Sep 23 15:11:07 redhat.test.com systemd[1]: proc-sys-fs-binfmt_misc.automount: Automount point already active?
         Sep 23 15:11:07 redhat.test.com systemd[1]: proc-sys-fs-binfmt_misc.automount: Got automount request for /proc/sys/fs/binfmt_mis
 
     Examples:
-        >>> "Automount point already active?" in log
+        >>> type(systemctl_status)
+        <class 'insights.parsers.systemctl_status_all.SystemctlStatusAll'>
+        >>> "Automount point already active?" in  systemctl_status
+        True
+        >>> systemctl_status.state == 'degraded'
+        True
+        >>> systemctl_status.jobs == '0 queued'
+        True
+        >>> systemctl_status.failed == '2 units'
         True
     """
-    pass
+    _state = _jobs = _failed = False  # Use False to avoid conflicting with None
+
+    def __get_item(self, key):
+        ret = self.get(key, num=1)
+        if ret:
+            return ret[0]['raw_message'].split(':')[1].strip()
+        return None
+
+    @property
+    def state(self):
+        """
+        (str): "State" shown in the command output, `None` by default.
+        """
+        if self._state is False:
```

Comment:
Here comes a volunteer for code reviewing.  😆 

I can see the usage of class attribute `_state = False` is a pretty economic choice here.
I am good with the code.

Reply:
Thanks for review @JoySnow